src/slider/interfaces.ts

```diff
@@ -0,0 +1,70 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import { BaseComponentProps } from '../internal/base-component';
+import { FormFieldValidationControlProps } from '../internal/context/form-field-context';
+import { NonCancelableEventHandler } from '../internal/events';
+
+export interface SliderProps extends BaseComponentProps, FormFieldValidationControlProps {
+  /**
+   * Indicates the current value.
+   */
+  value?: number;
+
+  /**
+   * Indicates the minimum value.
+   */
+  min: number;
+
+  /**
+   * Indicates the maximum value.
+   */
+  max: number;
```

Comment:
What happens to the component when min is greater than max?

Reply:
It should not be valid. Shall we throw a warning?

Reply:
I think we should use warnOnce but also provide some reasonable fallback